src/App.js

```diff
@@ -1,8 +1,13 @@
+import { Routes, Route } from 'react-router-dom';
+import Booklist from './components/Booklist';
+import Bookstate from './components/Bookstate';
+
 function App() {
   return (
```

Comment:
- [ ] I kindly suggest that you import and use your Navigation component into your App component so that it'll be possible to navigate between the home page and categories.